burgerNode.js

```diff
@@ -3,11 +3,11 @@ const BurgerTransaction = require('./burgerTransaction');
 const BurgerMiner = require("./burgerMiner");
 const BurgerWallet = require('./burgerWallet');
 const uuidv4 = require('uuid/v4');
-
+ 
 class BurgerNode {
     constructor(burgerBlockchain, configurations) {
         this.chain = burgerBlockchain;
-        this.nodes = [];
+        this.nodes = {};
```

Comment:
@24thsaint won't this cause issues with concurrent branches that are being developed, such as mine?

Reply:
No. Tried to merge master, yours, and mine - everything is alright with minimal merge conflicts in `BurgerBlockchain` and `server`.